src/atomicgen.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
Is this correct?

Reply:
Yes, you are never supposed to run from system python directly.  This runs with the user's python, which may be system one, but doesn't have to be.